configure.ac

```diff
@@ -3,7 +3,7 @@ AC_INIT([pick], [1.4.0], [hello@thoughtbot.com])
 AM_INIT_AUTOMAKE([subdir-objects])
 AC_CONFIG_HEADERS([config.h])
 AC_PROG_CC
-AC_CHECK_HEADERS([locale.h stdlib.h string.h unistd.h])
+AC_CHECK_HEADERS([limits.h locale.h stdlib.h string.h unistd.h])
```

Comment:
The `sys/limits.h -> limits.h` change makes sense and is more
portable. However, I would not bother adding the extra check since
limits.h is defined by POSIX[1].

[1] http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/limits.h.html